config/locales/es/admin.yml

```diff
@@ -620,6 +620,16 @@ es:
         add: Añadir como gestor
       search:
         title: "Gestores: Búsqueda de usuarios"
+    sdg_managers:
+      index:
+        title: Gestores ODS
+        name: Nombre
+        email: Email
+        no_sdg_managers: No hay gestores ODS.
+      manager:
+        add: Añadir como gestor ODS
```

Comment:
There's a missing translation error here :thinking:. The expected key is `admin.sdg/managers.sdg/manager.add`.